api.planx.uk/modules/pay/service/inviteToPay/sendConfirmationEmail.ts

```diff
@@ -47,7 +46,12 @@ async function getDataForPayeeAndAgentEmails(
         flow {
           slug
           team {
-            notifyPersonalisation: notify_personalisation
+            notifyPersonalisation: team_settings {
```

Comment:
Where queries are using a custom type (not a shared `Team` type from planx-core), I've kept the contextual variable name previously used to minimise changes in this already noisy PR.